app/src/main/java/com/sci/recipeandroid/feature/personalize/domain/service/PersonalizeService.kt

```diff
@@ -0,0 +1,96 @@
+package com.sci.recipeandroid.feature.personalize.domain.service
+
```

Comment:
please use dependency inversion . in service , repository , datasource layers

Reply:
Updated in e2ceaf1519d2ad526bf9d419752063a820c4ad13